bcipy/tasks/rsvp/copy_phrase.py

```diff
@@ -44,368 +71,610 @@ class RSVPCopyPhraseTask(Task):
     """
 
     TASK_NAME = 'RSVP Copy Phrase Task'
-
-    def __init__(
-            self, win, daq, parameters, file_save, signal_model, language_model, fake):
+    PARAMETERS_USED = [
```

Comment:
What's this list for?

Reply:
I'm guessing you're thinking of doing things like:
```python
t = Task(...)
...
for p in t.PARAMETERS_USED:
    add_display_widget(p)
```

But it looks like this is currently unused (or am I missing something?)

Reply:
Good call out. Because so many parameters are used for initializing the copy phrase wrapper and the display it took a while to figure out what parameters were needed for my unit tests. I intended to do a quick assertion here that all required parameters were provided but forgot to include it. I'll add that code.